project_euler/problem_02/sol5.py

```diff
@@ -0,0 +1,46 @@
+"""
```

Comment:
Please edit the filename to be `problem_05/sol5.py` so that it is in the appropriate directory.